### PR TITLE
[EAGLE-934] eagle 0.5 assembly update

### PR DESCRIPTION
**Pull request by jhsenjaliya**

### CODE REVIEW

**Comment by jhsenjaliya** [comment]
@haoch, can u please help reviewing this ?

File: eagle-assembly/src/main/conf/eagle.conf [hunk 1]
```diff
@@ -75,7 +77,8 @@ metadata {
     username = "root"
     password = ""
     driverClassName = com.mysql.jdbc.Driver
-    url = "jdbc:mysql://server.eagle.apache.org:3306/eagle"
+    database = "eagle"
+    connection = "jdbc:mysql://server.eagle.apache.org:3306/eagle?user=root"
```
Comment (haoch) [inline eagle-assembly/src/main/conf/eagle.conf]: Is the `user=root` duplicated with `username` field?

File: eagle-server-assembly/src/main/bin/eagle-server.sh [hunk 2]
```diff
@@ -1,136 +0,0 @@
-#!/bin/bash
```
Comment (haoch) [inline eagle-server-assembly/src/main/bin/eagle-server.sh]: Do you delete `eagle-env.sh` and `eagle-server.sh`?

**Comment by haoch** [comment]
@jhsenjaliya The shell scripts seems no correct

File: eagle-assembly/src/main/conf/eagle.conf (hunk 1 above)
Comment (jhsenjaliya) [inline eagle-assembly/src/main/conf/eagle.conf]: ya, i noticed that too, but kept whatever was there, let me remove it from connection

**Comment by haoch** [comment]
LGTM

**Pull request closed without merging**